sdk/storage/azure-storage-files-datalake/test/datalake_directory_client_test.cpp

```diff
@@ -382,6 +383,8 @@ namespace Azure { namespace Storage { namespace Test {
       auto anonymousClient
           = Azure::Storage::Files::DataLake::DirectoryClient(directoryClient.GetUri());
 
+      std::this_thread::sleep_for(std::chrono::seconds(30));
```

Comment:
Make sure this change goes to master first and rebase to avoid smuggling this fix here

Reply:
The same fix elsewhere was already made in master in this PR: https://github.com/Azure/azure-sdk-for-cpp/pull/1095. I am just re-applying it to other places that were missed. To me, having it here helps link the reason that exposed the issue, which is how winhttp handles connection re-use, specifically in this PR.

As an aside, running the storage live tests takes a long time (in CI), and it seems wasteful to run such tests multiple times for the sake of PR isolation, in this case, given the age of this PR. I would rather not have to wait :)

Reply:
So, as soon as you merge latest master to your branch, this changes should disappear right?